### PR TITLE
release_notes: add macaroons

### PR DESCRIPTION
**Pull request by guggero**
Forgot to add the release notes for the macaroons in #281.

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -15,4 +15,8 @@ This file tracks release notes for the loop client.
 
 #### Breaking Changes
 
+* Macaroon authentication has been enabled for the `loopd` gRPC and REST
```
Comment (carlaKC) [inline release_notes.md:18]: Perhaps we want to mention that it will pick up by default on mainnet, but testnet needs the flag?

**Review by carlaKC — APPROVED** [review]
LGTM! 

Not sure how much detail with these notes (can discuss as a team sometime), but I think it's worth pointing out that testnet now needs the network flag in this case.  

> Forgot to add the release notes for the macaroons in #281.

PR template will have your back in the future!

File: release_notes.md (hunk 1 above)
Comment (joostjager) [inline release_notes.md:18]: Do you want to add why this (in combination with tls) is good?

Comment (joostjager) [reply release_notes.md:18]: I mean something about remote node support

Comment (guggero) [reply release_notes.md:18]: Okay, I added the comment about the network. The rest (what directory things are located and so on) is described in the linked documentation so I think that should be fine.

Comment (guggero) [reply release_notes.md:18]: Added a comment about exposing the port to the internet.

**Review by joostjager — APPROVED** [review]

File: release_notes.md (hunk 1 above)
Comment (joostjager) [reply release_notes.md:18]: I think this was already the case for 0.9.0.

**Pull request merged**